controllers/stocksController.js

```diff
@@ -0,0 +1,44 @@
+const stocks = require('../models/stocks')
+/**
+ * Creates new stock
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Stock object
+ * @param res {Object} - Express response object
+ */
+const addNewStock = async (req, res) => {
+  try {
+    const { id, stockDetails } = await stocks.addStock(req.body)
+    return res.json({
+      message: 'Stock created successfully!',
+      stock: stockDetails,
```

Comment:
No validation on `req.body`?

Reply:
We have middleware in place for validations before reaching out to controller.